File: app/src/main/java/com/chichiangho/base/activity/SplashActivity.java
package com.chichiangho.base.activity;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

import com.chichiangho.base.base.CrashHandler;
import com.chichiangho.base.utils.DeviceUtil;
import com.chichiangho.base.utils.PreferenceUtil;

/**
 * Created by chichiangho on 2017/5/18.
 */

public class SplashActivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        doInitThings();

        if (PreferenceUtil.getUser() != null) {
            startActivity(new Intent(this, MainActivity.class));
        } else {
            startActivity(new Intent(this, MainActivity.class));
        }
        finish();
    }

    private void doInitThings() {
        new CrashHandler().init();
        DeviceUtil.getInstance().init(this);
//        OkHttpClientManager.setCertificates(getAssets().open("证书"));
    }
}

File: app/src/main/java/com/chichiangho/base/activity/SplashActivity.kt
package com.chichiangho.base.activity

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.chichiangho.base.utils.Preference
import com.chichiangho.common.base.CrashHandler

/**
 * Created by chichiangho on 2017/5/18.
 */

class SplashActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        doInitThings()

        if (Preference.loginUser != null) {
            startActivity(Intent(this, DailianActivity::class.java))
        } else {
            startActivity(Intent(this, DailianActivity::class.java))
        }
        finish()
    }

    private fun doInitThings() {
        CrashHandler().init()
        //        OkHttpClientManager.setCertificates(getAssets().open("证书"));
    }
}
